platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/style/expressions/ExpressionTest.kt

```diff
@@ -455,6 +455,27 @@ class ExpressionTest {
         Assert.assertTrue("expression should match", Arrays.deepEquals(expected, actual))
     }
 
+    @Test
```

Comment:
I was going to also implement wrapper for iOS, but realised that it is not so straightforward for me: I am not very familiar with NSExpression/NSPredicate and how we use it for expressions(can only guess using other expressions as reference, but it is still not so easy), so decided to leave it for another PR (but if someone can guide me through it - I can try to add it as well).

Reply:
It would be great if you could open an issue for this and link this PR.

Reply:
Even though there is no wrapper, it can still be used with style JSON directly right?

Reply:
> Edit: Actually, we can just update the existing ticket.

Done.

> Even though there is no wrapper, it can still be used with style JSON directly right?

Tbh it is very hard to answer for me: I am not very familiar with current iOS API and what is possible there and what is not...